cmd/api/main.go

```diff
@@ -62,6 +62,7 @@ type ConfigFile struct {
 func main() {
 
 	viper.AddConfigPath("./env")
```

Comment:
Isn't this line redundant now?

Reply:
I couldn't verify an actual release (it's failing on issues fixed in code, but not yet released) which could differ from the deployment used for the e2e tests, so I fixed the e2e test deployment with the least amount of risk. Once there is a deployable release I can verify and remove that line if redundant.